core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSupervisor.scala

```diff
@@ -125,6 +133,15 @@ class SubmissionSupervisor(executionServiceCluster: ExecutionServiceCluster,
       saveGlobalJobExecCounts(submissionStatuses, workflowStatuses)
   }
 
+  private def scheduleInitialMonitorPass: Cancellable = {
+    //Wait anything _up to_ the poll interval for a much wider distribution of submission monitor start times when Rawls starts up
+    system.scheduler.scheduleOnce(addJitter(0 seconds, submissionPollInterval), self, StartMonitorPass)
+  }
+
+  private def scheduleNextMonitorPass: Cancellable = {
+    system.scheduler.scheduleOnce(addJitter(submissionPollInterval), self, StartMonitorPass)
```

Comment:
Just wondering if we want a separate config for this. `submissionPollInterval` defines how often we poll for a single submission, and is currently set to 1 minute. Should we be calling `listAllActiveSubmissions` less frequently? (I'm not sure how expensive that query is.)

Reply:
That's a good point. It looks like this `listAllActiveSubmissions` query returns way more info than we actually need. For all running submissions it will load: the method config, all of the workflows, the root entity, and the workspace. The only info that the monitor even uses are submission.id, workspace.name, and workspace.namespace. It doesn't look like it even needs the name and namespace (except for logging errors), so maybe workspace.id would suffice for logging and we can get this down to a simple `select ID, WORKSPACE_ID from SUBMISSION where STATUS in [active_statuses]`

Reply:
Actually it looks like metrics rely on workspace name/namespace so I won't screw with that bit at the moment, but it'll still be an improvement to not load all of the workflows.